src/scripts/scripts/zone/karazhan/boss_nightbane.cpp

```diff
@@ -457,6 +462,10 @@ struct boss_nightbaneAI : public ScriptedAI
                 m_creature->GetMotionMaster()->MovePoint(3,IntroWay[3][0],IntroWay[3][1],IntroWay[3][2]);
 
                 Flying = true;
+
+				// Set flying speed to x6.5 so doesnt take so long to land
+				m_creature->SetSpeed(MOVE_FLIGHT, 6.0f);
```

Comment:
Oh, I was doing tests on my PTR and did a last second edit, forgot to change the comment. Good find